test_plans/root_group.asciidoc

```diff
@@ -0,0 +1,118 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl_ext_oneapi_root_group
+
+This is a test plan for the APIs described in
+https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/proposed/sycl_ext_oneapi_root_group.asciidoc[sycl_ext_oneapi_root_group].
+
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_ONEAPI_ROOT_GROUP` so they can be skipped
+if feature is not supported.
+
+== Tests
+
+* All following tests run with `Dimensions` = 1, 2, 3
+* Get root group object by
+[source,c++]
+----
+auto bundle = sycl::get_kernel_bundle(q.get_context());
+auto kernel = bundle.get_kernel<class KernelName>();
+auto maxWGs = kernel.ext_oneapi_get_info<sycl::ext::oneapi::experimental::info::kernel_queue_specific::max_num_work_group_sync>(q);
+if (maxWGs < 1)
+  SKIP("maxWGs should be at least 1");
+auto nd_range = sycl::nd_range<Dimensions>{global_range, local_range};
+auto props = sycl::ext::oneapi::experimental::properties{sycl::ext::oneapi::experimental::use_root_sync};
+q.parallel_for<class KernelName>(nd_range, props, [=](sycl::nd_item<Dimensions> it) {
+    auto root = it.ext_oneapi_get_root_group();
+});
+----
+where `global_range.size()` = `local_range.size() * maxWGs`.
+
+`maxWGs` should be at least `1`, otherwise test for this `Dimensions` is skipped.
```

Comment:
Sorry, I meant that the test should fail if `maxWGs` is less than `1`.  The spec says that the value returned by `kernel_queue_specific::max_num_work_group_sync` must be at least `1`.

Reply:
Up to you.  I think this is fine.